apps/studio/esbuild.mjs

```diff
@@ -145,7 +145,12 @@ const electronRendererPlugin = {
           to: './dist/'
         },
       ]
-    })]
+    })],
+    // import.meta.url is undefined in esbuild
```

Comment:
What is this used for?

Reply:
This was used by the new package I added to `open` a externals url. I can revert this since we don't need the package anyway.